### ISSUE TITLE
TensorFlow 2: TensorBoard import error

### ISSUE THREAD

**Issue by pascal-hwky** [wandb/client#1160]
`wandb --version && python --version && uname`

* Weights and Biases version: 0.9.3
* Python version: 3.6.9
* Operating System: Linux

### Description

I am trying to sync the client with TensorBoard in a Docker container with the TF2.2 base image. However, the client is unable to import TensorBoard, although I can import this package just fine. [Related issue](https://github.com/wandb/client/issues/691).

### What I Did

I have created a minimal [repository](https://github.com/pascal-hwky/wandb-tb-bug) to reproduce the issue.

```
git clone https://github.com/pascal-hwky/wandb-tb-bug.git
cd wandb-tb-bug
./run.sh
```
Sample output:

```
wandb: Dry run mode, not syncing to the cloud.
wandb: Tracking run with wandb version 0.9.3

...

wandb: ERROR Couldn't import tensorboard, not streaming events. Run `pip install tensorboard`
wandb: Program ended successfully.
wandb: You can sync this run to the cloud by running: 
wandb: wandb sync wandb/dryrun-20200716_123910-351imp5c
```

Any help is appreciated, as I cannot currently use my favorite experiment tracking framework with the Object Detection API. Running in Docker is a requirement for me, I cannot use virtual environments as suggested in the [previous issue](https://github.com/wandb/client/issues/691#issuecomment-648478967).

**Comment by tyomhak** [comment]
Hi there, thank you for reporting. I'm filing an internal ticket for this. We'll look into this and get back to you.

**Comment by pascal-hwky** [comment]
Hi @tyomhak, do you have any update on this issue? Were you able to reproduce it?

**Comment by ktobah** [comment]
Any updates on this issue? I have it with tensorflow 2.3.0 too.

**Comment by vanpelt** [comment]
Apologies for the delay.  The issue is being caused by `sys.path` issue messing up our ability to import tensorboard.  We've fixed this in our latest library.  You can install it with `pip install wandb -U --pre`.  I've verified this works in the example docker image provided.

**Comment by pascal-hwky** [comment]
Can confirm this is fixed in version 0.10.2.
